packages/core-flows/src/product/workflows/create-product-variants.ts

```diff
@@ -1,6 +1,14 @@
 import { ProductTypes } from "@medusajs/types"
-import { WorkflowData, createWorkflow } from "@medusajs/workflows-sdk"
-import { createProductVariantsStep } from "../steps"
+import {
+  WorkflowData,
+  createWorkflow,
+  transform,
+} from "@medusajs/workflows-sdk"
+import {
+  createProductVariantsStep,
+  createVariantPricingLinkStep,
+} from "../steps"
+import { createPriceSetsStep } from "../../pricing"
 
 type WorkflowInput = {
   product_variants: ProductTypes.CreateProductVariantDTO[]
```

Comment:
suggestion: would be good to add a specific workflow type here since we're accepting prices here along with `CreateProductVariantDTO`